packages/types/src/market.ts

```diff
@@ -40,3 +40,47 @@ export type FindAllMarketArgs = {
 export type MarketCapResult = {
   [k: string]: MarketData
 }
+
+/**
+ * stackedQ: We need the list in the frontend,
+ * and there's no way to convert an union type to an array,
+ * so the list is defined as an array of strings,
+ * but SupportedFiatCurrencies is an union of strings
+ */
+export const SupportedFiatCurrenciesList = [
+  'CNY',
+  'USD',
+  'EUR',
+  'JPY',
+  'GBP',
+  'KRW',
+  'INR',
+  'CAD',
+  'HKD',
+  'AUD',
+  'TWD',
+  'BRL',
+  'CHF',
+  'THB',
+  'MXN',
+  'RUB',
+  'SAR',
+  'SGD',
+  'ILS',
+  'IDR'
+] as const
+
+export type SupportedFiatCurrencies = typeof SupportedFiatCurrenciesList[number]
+
+export type FiatMarketDataArgs = {
+  symbol: SupportedFiatCurrencies
+}
+
+export type FiatPriceHistoryArgs = {
+  symbol: SupportedFiatCurrencies
+  timeframe: HistoryTimeframe
+}
+
+export type FindByFiatMarketType = (args: FiatMarketDataArgs) => Promise<MarketData | null>
+
+export type FiatPriceHistoryType = (args: FiatPriceHistoryArgs) => Promise<HistoryData[]>
```

Comment:
I don't think any of these need to be in the `types` package.  The only types that should be in `types` are ones that are used in multiple lib packages to avoid circular dependencies.

These types look to be very `market-service` specific and aren't imported into any other packages.

Move these to the `market-service` package.